packages/p2p/src/components/my-ads/create-ad-form.jsx

```diff
@@ -406,7 +407,12 @@ const CreateAdForm = () => {
                                                 className='p2p-my-ads__form-button'
                                                 primary
                                                 large
-                                                is_disabled={isSubmitting || !isValid || !selected_methods.length}
+                                                is_disabled={
+                                                    isSubmitting ||
+                                                    !isValid ||
+                                                    !selected_methods.length ||
+                                                    my_ads_store.current_method.is_deleted
```

Comment:
if the user is in the middle of deleting a pm (clicking the close circle icon), need to disable the button so users can't post the ad